res/database/Default/database_news.xml

```diff
@@ -1556,11 +1556,11 @@ Demonstrations are expected again this year.</en>
 		<news id="2dd29a642-a0dc-f275-32ef-16815eb34ff" thread_id="9707f767-9129-46ea-84d2-a4870897fdfb" type="2" creator="">
 			<title>
 				<de>Wildschwein-Urteil gefällt</de>
-				<en>Wild boar judgement adjudicated</en>
+				<en>Wild boar judgement passed</en>
 			</title>
 			<description>
 				<de>Die Bachenhausener Strafkammer befand den Gärtner Rico Mielke der Körperverletzung schuldig. „Sie sind wohl blind?“, fragte der klagende Anwalt.</de>
-				<en>The Bachenhausen criminal court found the gardener Rico Mielke guilty of assault. "I suppose you're blind?" asked the plaintiff's lawyer.</en>
+				<en>The Bachenhausen Criminal Court found gardener Rico Mielke guilty of assault. "You must be blind?" asked the plaintiff lawyer.</en>
```

Comment:
Are you blind?
question mark looks strange otherwise

Reply:
I think both is fine, but I'm indifferent, so ack.